arm/README.md

```diff
@@ -4,99 +4,100 @@ In this section you can find useful information regarding the Modules that are c
 
 | Name | Provider namespace | Resource Type |
```

Comment:
Undo updates - will be handled by the CI environment automatically.

Reply:
done

Reply:
Copy/paste upstream file

Reply:
@ArielRam99 don't forget to 'undo' this in a branch - not main. In main it will continue to update the table